### PR TITLE
PE-6751: auto-update manifest

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
--- Releases ---
Android release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:android:6cf0cd5ec064fad3ffce07/releases/4afd6gbg22d2g

### CODE REVIEW

File: lib/blocs/upload/upload_cubit.dart [hunk 1]
```diff
@@ -88,6 +97,147 @@ class UploadCubit extends Cubit<UploadState> {
...
+        completedCount: ++completedCount,
+      ));
+    }
+
+    emit(UploadComplete(
+      manifestFiles: _selectedManifestFiles,
+    ));
+  }
+
+  void cancelManifestsUpload() {
```
Comment (karlprieb) [inline lib/blocs/upload/upload_cubit.dart:234]: Is there any code that calls `cancelManifestsUpload`?

File: lib/blocs/upload/upload_cubit.dart [hunk 2]
```diff
@@ -1100,7 +1269,13 @@ class UploadCubit extends Cubit<UploadState> {
...
         );
 
-        emit(UploadComplete());
+        if (_selectedManifestFiles.isNotEmpty) {
+          await prepareManifestUpload();
+        }
+
+        // emit(UploadComplete(
+        //   manifestFiles: _selectedManifestFiles,
+        // ));
```
Comment (karlprieb) [inline lib/blocs/upload/upload_cubit.dart:1278]: Can you remove this commented block?

File: lib/components/upload_form.dart [hunk 3]
```diff
@@ -519,49 +660,389 @@ class _UploadReadyModalBaseState extends State<UploadReadyModalBase> {
...
+                                            child: action,
+                                          ),
+                                        );
+                                      }).toList(),
+                                    ),
+                                  )
+                                ],
+                              ),
+                            ),
+                            // TODO: add back
```
Comment (karlprieb) [inline lib/components/upload_form.dart:937]: When?

File: lib/blocs/upload/upload_cubit.dart (hunk 1 above)
Comment (thiagocarvalhodev) [reply lib/blocs/upload/upload_cubit.dart:238]: Yes. On upload_form.dart

File: lib/components/upload_form.dart (hunk 3 above)
Comment (thiagocarvalhodev) [reply lib/components/upload_form.dart:933]: Next ticket we'll implement the UI for mobile

**Review by karlprieb — APPROVED** [review]

**Review by atticusofsparta — APPROVED** [review]

**Review by karlprieb — APPROVED** [review]

**Pull request merged**